arangod/RestServer/FlushFeature.cpp

```diff
@@ -59,6 +460,123 @@ void FlushFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      arangodb::options::makeFlags(arangodb::options::Flags::Hidden));
 }
 
+/*static*/ bool FlushFeature::registerFlushRecoveryCallback(
```

Comment:
please explicitly specify that `registerFlushRecoveryCallback` is not thread-safe

Reply:
that is specified in the header declaration of the function